README.md

```diff
@@ -0,0 +1,117 @@
+To start the local development server your system requires [PHP](https://www.php.net/manual/en/install.php) and [Symfony](https://symfony.com/download):
+```
+brew install php
+brew install symfony-cli/tap/symfony-cli
+```
+
+You then need to install [Composer](https://getcomposer.org/download/):
+```
+curl -sS https://getcomposer.org/installer | php
+```
+With Composer you should be able to install all required packages for this project:
+```
+sudo -E env "PATH=$PATH" php composer.phar install
+```
+
+You should now be able to run this demo by typing `symfony server:start` into your console while being located in the root directory of this repository. If the setup was done correctly, the following messages should appear in your terminal:
+![image](https://user-images.githubusercontent.com/23581140/205909459-7ed3d679-b313-40d3-85be-1178b80a1594.png)
+To verify that your instance is running without errors enter `http://localhost:8000/ping` into your browser. If "pong" is displayed, you can continue with the next step.
+
+#### 2.2.2. Ngrok
+
+The endpoints of your local system have to be public so Corbado can send requests there. To make your local instance publicly availbale we use [ngrok](https://ngrok.com/download) which is a reverse proxy service. It assigns you a globally availably URL and forwards all incoming requests to your local instance. It can be installed using:
+```
+curl -s https://ngrok-agent.s3.amazonaws.com/ngrok.asc | sudo tee /etc/apt/trusted.gpg.d/ngrok.asc >/dev/null && echo "deb https://ngrok-agent.s3.amazonaws.com buster main" | sudo tee /etc/apt/sources.list.d/ngrok.list && sudo apt update && sudo apt install ngrok
+```
+
+You can start your ngrok instance by typing `ngrok http 8000`. In your terminal you should see the following:
+![image](https://user-images.githubusercontent.com/23581140/205919914-986f95ea-7c32-4501-a651-f47b16e3b2e2.png)
+
+Entering the URL which is inside the red rectangle with `/ping` as path (In our case `https://d15e-212-204-96-162.eu.ngrok.io/ping`) should now display "pong" as well since this ngrok URL just forwards requests to your local instance.
+
+
+### 2.3. Corbado developer panel settings
+
+>**Warning**
+>Remember to always press "Save changes" after entering details in the Corbado developer panel!
+>
+#### 2.3.1. Configure CNAME
+
+In the developer panel under `Project settings -> Web component` enter the CNAME you previously created. 
+>**Warning**
+>It can take up to 5 minutes until our system has registered your CNAME
+
+![image](https://user-images.githubusercontent.com/23581140/205950309-f6f622e5-94ca-4413-9384-d7a2605da75d.png)
+
+#### 2.3.2. Authorize origins
+Inside the developer panel under `Project settings -> REST API` you need to enter the following origins in order to allow them to share resources.
+1. `https://auth.your-company.com`, the CNAME which represents the Corbado web component
+2. `http://localhost:8000`, which is where your backend instance is running
```

Comment:
backend instance -> oben erklären oder hier was das konkret ist in unserem Beispiel